### DISCUSSION TITLE
Saving and loading objects and joints

### DISCUSSION THREAD

**Discussion by RamiManaf** [dyn4j/dyn4j#205] [Q&A] [answered]
Any idea on how to save objects and joints in a file?

**Comment by wnbittle** [comment, marked as answer]
The Sandbox application, while very old now and not up-to-date w/ the latest dyn4j, has a good example of how to [save/load using XML](https://github.com/dyn4j/dyn4j-sandbox/tree/master/src/main/java/org/dyn4j/sandbox/persist).
